Replicator/tests/ReplicatorCollectionTest.cc

```diff
@@ -511,6 +511,93 @@ TEST_CASE_METHOD(ReplicatorCollectionTest, "Multiple Collections Incremental Pus
     validateCollectionCheckpoints(db, db2, 1, "{\"local\":22,\"remote\":24}");
 }
 
+struct Jthread {
+    std::thread thread;
+    Jthread(std::thread&& thread_)
+    : thread(move(thread_))
+    {}
+    ~Jthread() {
+        thread.join();
+    }
+};
+
+TEST_CASE_METHOD(ReplicatorCollectionTest, "Multiple Collections Incremental Revisions", "[Push][Pull]") {
+    addDocs(db, Roses, 2, "db-Roses-");
+    addDocs(db, Tulips, 2, "db-Tulips-");
+    C4Collection* roses = c4db_getCollection(db, Roses, nullptr);
```

Comment:
There is `getCollection(db, Roses)` small util that also checks if the returned collection exists.

Reply:
I saw it. Thanks